common/sai_data.py

```diff
@@ -1,126 +1,18 @@
 import json
 from enum import Enum
+from sai_thrift.sai_headers import *
```

Comment:
I think this will break Redis case.

Reply:
Good point!
I'll think about how to do it differently. I'll appreciate it if you have any suggestions.

Reply:
As we discussed earlier, SAI metadata has to be used instead
https://github.com/opencomputeproject/SAI-Challenger/blob/2d58df2fe75eab5246342bf1689c887a95b22a3f/common/sai.py#L179
One approach for both Redis and Thrift flows

Reply:
Well... actually not sure whether the list of object types is in metadata. We should check. That should be the right approach.

Reply:
As I understand there is already a task in the backlog to fix this for sai_thrift_metadata.py.
Maybe to keep things clear the best way to resolve this is to:
1. Keep SaiObjType and SaiStatus as constant Enums for this PR
2. Switch to autogenerating in a separate PR for both sai_data.py and sai_thrift_metadata.py

Reply:
Both options are fine to me.

Reply:
Updated.